api/src/main/java/com/cloud/host/Status.java

```diff
@@ -119,45 +122,45 @@ public Status getNextStatus(Event e) throws NoTransitionException {
     static {
```

Comment:
if a true flag is passed at the end, that is the transitionAlertFlag.  If it is True then an alert should be raised.  If it is False or not passed at all, then an alert should NOT be raised.